wordpress-mobile/WordPress-iOS
# Activity Log: Integrate new rewind endpoint (remove old one)

**Another step on** #7832 

The new `rewind` endpoint provides information about the status of rewind on the site, as well as the last restore operation performed.
In this PR we are only using it for the latter.

**To test:**
1. Go to the Activity Log
2. Do a rewind, check that everything looks as expected.

CC: @dmsnell
